samples/hello-spring-boot/build.gradle.kts

```diff
@@ -0,0 +1,37 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = "com.spring.boot"
+version = "1.0.0-SNAPSHOT"
+
+plugins {
+    val kotlinVersion = "1.2.30"
+    id("org.springframework.boot") version "2.0.0.RELEASE"
+    kotlin("jvm") version kotlinVersion
+    kotlin("plugin.spring") version kotlinVersion
+    id("io.spring.dependency-management") version "1.0.4.RELEASE"
+}
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions {
+        jvmTarget = "1.8"
+        freeCompilerArgs = listOf("-Xjsr305=strict")
+    }
+}
+
+(tasks.findByName("test") as Test).useJUnitPlatform()
```

Comment:
personally I find 
```
tasks.withType(Test::class.java) {
    useJUnitPlatform()
}

```

more idiomatic

Reply:
@breskeby
This is simpler:
```kotlin
tasks.withType<Test> {
    useJUnitPlatform()
}
```

But this is bikeshedding. The way it is here also makes sense.